build.xml

```diff
@@ -283,10 +286,21 @@
         </fileset>
     </path>
 
-    <target name="_build-accord">
-      <echo unless:set="env.ACCORD_LOCAL_DEV">building Accord, set ACCORD_LOCAL_DEV environment variable to skip</echo>
-      <exec unless:set="env.ACCORD_LOCAL_DEV" executable=".build/include-accord.sh" dir="${basedir}" logError="true" failonerror="true" failifexecutionfails="true" />
-      <echo if:set="env.ACCORD_LOCAL_DEV">ACCORD_LOCAL_DEV environment variable is set, skipping pull/install</echo>
+    <target name="_build-accord" depends="init">
+      <exec executable="${basedir}/accord/gradlew" dir="${basedir}/accord" logError="true" failonerror="true" failifexecutionfails="true">
+        <!-- Need to call clean as a version change with a jar that didn't have code changes will not procude a new jar with the new version... so need clean to make sure build is stable -->
+        <arg value="clean" />
+        <arg value="build" />
+        <arg value="copyDependencies" /> <!-- For IDE support -->
+        <arg value="publishToMavenLocal" />
+        <arg value="-x" />
+        <arg value="test" />
+        <arg value="-x" />
+        <arg value="rat" />
+        <arg value="-Paccord_group=org.apache.cassandra" />
+        <arg value="-Paccord_artifactId=cassandra-accord" />
+        <arg value="-Paccord_version=${version}" />
+      </exec>
```

Comment:
nit: what about moving all accord ant xml stuff into `.build/build-accord.xml` ? 

(there's two things happening in this PR: 1. git submodule stuff, and 2. accord stuff)